superset-frontend/src/dashboard/actions/sliceEntities.ts

```diff
@@ -114,13 +108,11 @@ export function fetchSlices(
       ? [{ col: 'slice_name', opr: 'chart_all_text', value: filter_value }]
       : [];
 
-    if (isFeatureEnabled(FeatureFlag.DASHBOARD_NATIVE_FILTERS)) {
-      filters.push({
-        col: 'viz_type',
-        opr: 'neq',
-        value: 'filter_box',
-      });
-    }
+    filters.push({
```

Comment:
Do we still need this logic given that filter boxes are slatted to be removed in https://github.com/apache/superset/pull/26328?

Reply:
Although related, the features work independently which allows us to remove them in separate PRs. Anything related to filter boxes will be removed in a specific PR.